api/annotations.go

```diff
@@ -154,6 +152,11 @@ const (
 	certificateAnnotationKeyProviderCredentialSecretName = "certificate.appscode.com/provider-secret"
 	certificateAnnotationKeyACMEUserSecretName           = "certificate.appscode.com/user-secret"
 	certificateAnnotationKeyACMEServerURL                = "certificate.appscode.com/server-url"
+
+	// Sticky session configures proxy to use sticky connection
```

Comment:
This description is incomplete. Make it clear how this works when applied to a Ingress vs applied to a Service. Also the example is incorrect when used in JSON.

`ingress.appscode.com/sticky-session: "true"`

Reply:
could we apply this annotation in both? i have removed effects of applying this annotations in ingress. Should this be valid still?